models/GPTModel.py

```diff
@@ -0,0 +1,123 @@
+# In this code we have used GPT-2 Model for classification of IMDB Dataset:
+
+# Code is as followed:
+
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import accuracy_score
+from transformers import GPT2Tokenizer, GPT2ForSequenceClassification, GPT2Config
+from sklearn.pipeline import Pipeline
+from sklearn.base import BaseEstimator, TransformerMixin
+import torch
+import re
+
+
+# Defining cleaning function:
+def clean(text):
+    for token in ["<br/>", "<br>", "<br>"]:
+        text = re.sub(token, " ", text)
+    
+    text = re.sub("[\s+\.\!\/_,$%^*()\(\)<>+\"\[\]\-\?;:\'{}`]+|[+——！，。？、~@#￥%……&*（）]+", " ", text)
+    
+    return text.lower()
```

Comment:
Given that this clean is being re-used everywhere, do you want to refactor this out into a separate file?